src/Button.js

```diff
@@ -0,0 +1,31 @@
+import React, { useState } from "react";
+import "./index.css";
+
+import dice from "./images/icon-dice.svg";
+
+const Button = ({ handleClick }) => {
+	const [hover, setHover] = useState(false);
+
+	const handleMouseEnter = () => {
```

Comment:
As we discussed briefly in the interview, this is quite over-engineered. 
It is very well done though! Good job on implementing that 👍🏽 
However, I would recommend to rewrite it using a regular and popular pseudo-class selector. ✅